### PR TITLE
Ldanzinger/buffer

### PR DESCRIPTION
**Pull request by ldanzinger**
@michael-tims please review/merge

### CODE REVIEW

File: ArcGISRuntimeSDKQt_CppSamples/Geometry/Buffer/Buffer.cpp [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+#include "SimpleMarkerSymbol.h"
+#include "Envelope.h"
+#include "Viewpoint.h"
+#include "SpatialReference.h"
+
+using namespace Esri::ArcGISRuntime;
+
+Buffer::Buffer(QQuickItem* parent /* = nullptr */):
+  QQuickItem(parent),
+  m_bufferSize(5)
```
Comment (michael-tims) [inline ArcGISRuntimeSDKQt_CppSamples/Geometry/Buffer/Buffer.cpp]: Initialize in the header instead.
`int m_bufferSize = 5;`

File: ArcGISRuntimeSDKQt_CppSamples/Geometry/Buffer/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Buffer
+This sample demonstrates how to use the GeometryEngine::Buffer to generate a polygon of a fixed distance around a point.
```
Comment (michael-tims) [inline ArcGISRuntimeSDKQt_CppSamples/Geometry/Buffer/README.md]: Should `GeometryEngine::Buffer` be `GeometryEngine::buffer method`?  Not sure what we do in other places.

File: ArcGISRuntimeSDKQt_CppSamples/Geometry/Buffer/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+![Image](screenshot.png)
+
+## How to use the sample
+Tap on the map to create center map-points to generate buffer(s). You can optionally change the buffer distance (in miles) by adjusting the value in the text box before each tap on the map.
+
+## How it works
+The static method, `GeometryEngine.buffer`, is called by passing in a Point and a distance. The unit of measurement for the distances is automatically set to be the same as the input geometries, which in this case is meters. To provide a better user experience, the units entered in the User Interface (UI) for generating the buffer is in miles, so the provided number must be multiplied by 1609.34, which is the number of meters in a mile.
+
+## Revlevant API
```
Comment (michael-tims) [inline ArcGISRuntimeSDKQt_CppSamples/Geometry/Buffer/README.md]: Relevant

File: ArcGISRuntimeSDKQt_CppSamples/Geometry/Buffer/README.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+ - GeometryEngine.Buffer
```
Comment (michael-tims) [inline ArcGISRuntimeSDKQt_CppSamples/Geometry/Buffer/README.md]: Again, not sure if the `B` should be `b` here?

Comment (michael-tims) [reply ArcGISRuntimeSDKQt_CppSamples/Geometry/Buffer/README.md]: Same comments for QML files below.

**Comment by ldanzinger** [comment]
thanks @michael-tims. updated

File: ArcGISRuntimeSDKQt_QMLSamples/Geometry/Buffer/README.md [hunk 5]
```diff
@@ -12,7 +12,7 @@ Tap on the map to create center map-points to generate buffer(s). You can option
 The static method, `GeometryEngine.buffer`, is called by passing in a Point and a distance. The unit of measurement for the distances is automatically set to be the same as the input geometries, which in this case is meters. To provide a better user experience, the units entered in the User Interface (UI) for generating the buffer is in miles, so the provided number must be multiplied by 1609.34, which is the number of meters in a mile.
 
 ## Revlevant API
```
Comment (michael-tims) [inline ArcGISRuntimeSDKQt_QMLSamples/Geometry/Buffer/README.md]: @ldanzinger Relevant here as well.

**Comment by ldanzinger** [comment]
whoops, fixed now

**Pull request merged**